libs/core-scanner/src/pages/robots-txt.ts

```diff
@@ -72,7 +72,7 @@ const findRobotsCrawlDelay = (
   robotsTxt: string,
 ) => {
```

Comment:
To be clear what the return value is on this function, it may be useful to annotate the expected return type.

```suggestion
): number | null => {
```